content/post/70-putting-npm-on-ipfs-part-1.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2019-02-17
+title: Putting npm on IPFS Part 1 - The Registry
+author: Alex Potsides
+---
+
+npm is the de facto package manager for the JavaScript ecosystem and is largest registry in the world, with more than [900k](https://replicate.npmjs.com/_all_docs) packages and over 7 billion downloads a week. In recent years it's got a lot faster and more reliable, but what if we were to take it to the next level and put it on the distributed web?
```

Comment:
It'd might be good to explain _why_ putting it on the distributed web takes it to the next level, what benefits does it bring to compared to using the npm registry?

Reply:
Done!